airbyte-integrations/connectors/source-google-directory/source_google_directory/spec.json

```diff
@@ -0,0 +1,22 @@
+{
+  "documentationUrl": "https://docsurl.com",
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Google Directory Spec",
+    "type": "object",
+    "required": ["credentials_json", "credentials_pickle"],
+    "additionalProperties": false,
+    "properties": {
+      "credentials_json": {
+        "type": "string",
+        "description": "Path to the JSON credentials file.",
+        "airbyte_secret": true
+      },
+      "credentials_pickle": {
+        "type": "string",
+        "description": "Path to the pickle credentials file.",
```

Comment:
nitpick: I am not sure, but maybe we should add more extensive field description, or add link to Google Directory documentation, where can describe in detail what kind of object it is and how to generate it? WDYT

Reply:
This spec is incomplete for now since we don't provide oauth in Airbyte yet (but will soon). We don't expect the user to provide a credentials pickle, so we'll need to change this section once we see if svc accounts can be used (or once we enable oauth in airbyte, we'll need to implement the oauth flow in this connector)